src/Schema/SimulationPlotSchema_0.0.1.json

```diff
@@ -4,52 +4,145 @@
   "type": "object",
   "description": "AsyncApi Api Simulator. Define routes and settings regarding traffic Simulated",
   "patternProperties": {
-    "^group-[\\w\\d]+$": {
-      "type": "object",
-      "additionalProperties": false,
-      "properties": {
-        "eps": {
-          "description": "Events per second being emitted",
-          "type": "integer"
-        },
-        "priority" : {
-          "type": "integer",
-          "description": "This determines whether it should be prioritized in case the resources to emit events is insufficient."
-        }
+    "^scenario-[\\w\\d]+$" : {
+      "type": "array",
+      "items": {
```

Comment:
I would highly recommend start splitting up the schema file into digestible chunks, cause these are really quickly becoming unreadable 🙂

Reply:
i will split it with subschemas in the same file. there is some weird issue with json schema, it is not able to resolve external schemas.